backend/sozisel/test/sozisel/session_resource_links_test.exs

```diff
@@ -0,0 +1,100 @@
+defmodule Sozisel.SessionResourceLinksTest do
+  use Sozisel.DataCase
+
+  alias Sozisel.Model.{SessionResourceLinks, SessionResourceLinks.SessionResourceLink}
+
+  import Sozisel.Factory
+
+  describe "session_resource_links" do
+    setup do
+      user = insert(:user)
+      session = insert(:session, %{user_id: user.id})
+      [user: user, session: session, session_resource: insert(:session_resource)]
+    end
+
+    def session_resource_link_fixture(%{session_resource: session_resource, session: session}) do
+      {:ok, session_resource_link} =
+        %{}
+        |> Enum.into(%{is_public: true})
```

Comment:
Why did you event start with this empty map with all those places :P Just make it a single map as you have a single key nevertheless. Without all those `put` or `Enum.into` which are unnecessary.